### PR TITLE
On branch edburns-msft-78-contact-me-verbiage Extract current verbiage

### PR DESCRIPTION
**Pull request by edburns**
Your branch is up to date with 'origin/edburns-msft-78-contact-me-verbiage'.
deleted:    arm-oraclelinux-wls/src/main/resources/description.html
modified:   arm-oraclelinux-wls/src/main/resources/README.md

Signed-off-by: Ed Burns <edburns@microsoft.com>

### CODE REVIEW

**Comment by edburns** [comment]
Hello @rosemarymarano  and @mriccell , this README is our way to allow interested parties to control the text of the [CONTACT ME offer](https://aka.ms/azurewls). When the PR is merged, I will update Partner Center with the text in the README so that when the offer is re-published, the changes will appear.

**Comment by edburns** [comment]
Please give your editorial suggestions, I will apply them, and we will go through the normal review process.

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 1]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+Oracle WebLogic Server on Azure Solution Overview
+
+## Search results summary
+
+Oracle WebLogic Server is a scalable, enterprise-ready Java application server.
+
+## Description (Observed max 2973 characters, not including formatting)
+
+<p>Azure Oracle WebLogic Server (WLS) offers enable you to embrace cloud computing by making it as easy as possible to migrate your Java applications to Azure. The offers increase productivity by automating most boilerplate Java and Azure resource provisioning and configuration tasks so that you can focus on application development. The offers cover deployment to both Azure Virtual Machines and the Azure Kubernetes Service (AKS).</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:88]: have we not been calling it Oracle WebLogic Server on Azure offers?

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 2]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p><b>WLS on Virtual Machines</b><br/>
+The WLS on virtual machines offers automate provisioning virtual network, storage, and Linux resources, installing WLS, setting up security with a network security group, easing database connectivity, configuring load-balancing with App Gateway or Oracle HTTP Server, connecting to Azure Active Directory, enabling centralized logging via ELK as well as integrating distributed caching with Oracle Coherence.</p>
+
+<p>There are several offers that target use cases such as single node with admin server enabled and WLS cluster (including dynamic cluster). These offers supports a range of operating system, Java and WebLogic versions such as WLS 14 and JDK 11 on Oracle Linux 7.6 through base images. These base images are also available on Azure on their own. The base images are suitable for customers that require very highly customized Azure deployments. The current set of base images are available <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=oracle%20weblogic%20base%20image&amp;page=1">in the Azure marketplace</a>.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:99]: admin server -> Administration server
(including dynamic cluster) -> (including both configured and dynamic clusters)

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 3]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p><b>WLS on AKS</b><br/>
+The WLS on AKS offer automates provisioning an AKS cluster, the WebLogic Kubernetes Operator, WLS Docker images and the Azure Container Registry (ACR). The offer also supports configuring load balancing with Azure App Gateway or the Azure Load Balancer, easing database connectivity, publishing metrics to Azure Monitor as well as mounting Azure Files as Kubernetes Persistent Volumes. The offer will work with any WLS version that supports the Operator, such as 12.2.1.3 and 12.2.1.4.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:102]: WLS on AKS -> The WebLogic on AKS
such as 12.2.1.3 and 12.2.1.4 how about 14.1.1?

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 4]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p>Oracle and Microsoft also provide basic step-by-step guidance on <a href=” https://techcommunity.microsoft.com/t5/apps-on-azure/run-oracle-weblogic-server-on-azure-kubernetes-service/ba-p/1849719”>getting started with WLS and AKS</a>. This guidance is suitable for customers that wish to remain as close as possible to a native Kubernetes manual deployment experience.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:104]: WLS on AKS -> The WebLogic on AKS

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 5]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p>The offers are linked in the <i>Learn more</i> section at the bottom of this page.</p>
+
+<p>These offers are Bring-Your-Own-License. They assume you have already procured the appropriate licenses with Oracle and are properly licensed to run offers in Azure.</p>
+
+<p><i>If you want to work closely on your migration scenarios with the engineering team developing these offers, just hit the CONTACT ME button</i>. Program managers, architects and engineers will reach back out to you shortly and initiate collaboration!</p>
+
+<p><b>WLS on Virtual Machines</b><br/>
+The WLS on virtual machines offers automate provisioning virtual network, storage, and Linux resources, installing WLS, setting up security with a network security group, easing database connectivity, configuring load-balancing with App Gateway or Oracle HTTP Server, connecting to Azure Active Directory, enabling centralized logging via ELK as well as integrating distributed caching with Oracle Coherence.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:97]: WLS on virtual machines -> WebLogic on virtual machines

**Review by mriccell — CHANGES_REQUESTED** [review]

**Comment by edburns** [comment]
Applied changes in 1322525 .

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 6]
```diff
@@ -3,15 +3,152 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+   * PDF download of WLS_12_2_1_3_RELNOTES.pdf
+   
+## Videos
+
+* [WebLogic on Azure IaaS](https://www.youtube.com/watch?v=KZpG280G_vs)
 
-* [Learn more](http://www.oracle.com/technetwork/middleware/weblogic/learnmore/index.html)
+## Scratch not currently in offer 
 
 * [Documentation](https://wls-eng.github.io/arm-oraclelinux-wls/)
```
[the remark this answers is not in the document]
Comment (edburns) [reply weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:153]: Great catch. It is not.  It should be https://oracle.github.io/weblogic-azure/

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 7]
```diff
@@ -3,17 +3,154 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p><b>WebLogic Server on Virtual Machines</b><br/>
+The WebLogic Server on virtual machines offers automate provisioning virtual network, storage, and Linux resources, installing WLS, setting up security with a network security group, easing database connectivity, configuring load-balancing with App Gateway or Oracle HTTP Server, connecting to Azure Active Directory, enabling centralized logging via ELK as well as integrating distributed caching with Oracle Coherence.</p>
+
+<p>There are several offers that target use cases such as single node with Administration server enabled and WLS cluster (including both configured and dynamic clusters). These offers supports a range of operating system, Java and WebLogic versions such as WLS 14 and JDK 11 on Oracle Linux 7.6 through base images. These base images are also available on Azure on their own. The base images are suitable for customers that require very highly customized Azure deployments. The current set of base images are available <a href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps?search=oracle%20weblogic%20base%20image&amp;page=1">in the Azure marketplace</a>.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:99]: WLS 14 -> 14.1.1.0 
the images can be on Oracle Linux 7.6 or Oracle Linux 8.4

File: weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md [hunk 8]
```diff
@@ -3,17 +3,154 @@ Copyright (c) 2021, Oracle and/or its affiliates.
...
+
+<p><b>WebLogic Server on AKS</b><br/>
+The WebLogic Server on AKS offer automates provisioning an AKS cluster, the WebLogic Kubernetes Operator, WLS Docker images and the Azure Container Registry (ACR). The offer also supports configuring load balancing with Azure App Gateway or the Azure Load Balancer, easing database connectivity, publishing metrics to Azure Monitor as well as mounting Azure Files as Kubernetes Persistent Volumes. The offer will work with any WLS version that supports the Operator, such as 14.1.1, 12.2.1.3 and 12.2.1.4.</p>
```
Comment (mriccell) [inline weblogic-azure-vm/arm-oraclelinux-wls/src/main/resources/README.md:102]: such as 14.1.1 -> 14.1.1.0

**Comment by edburns** [comment]
Resolved in 44d9f20.

**Review by mriccell — APPROVED** [review]

**Pull request merged**